### ISSUE TITLE
Dropped into a location, immediately told I'm at the end of a route

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#1557]
Just logged in, started a new mission, and immediately received 'You have reached the end of this route.' Do we have a minimum street segment size to prevent this from happening?

![image](https://user-images.githubusercontent.com/1621749/54395589-716f2600-466d-11e9-9446-c9bd1b3b2e2b.png)

```
svl.panorama.getPano();
"DTdOw2a0VXo6GcZOy8Gzvw"
(svl.taskContainer.getCurrentTask()).getStreetEdgeId();
1482
```

I also clicked the 'Feedback' button and added a description of the problem.

**Comment by misaugstad** [comment]
>Do we have a minimum street segment size to prevent this from happening?

No we do not. And I think some tiny segments like that were added in when we split up the streets into our custom neighborhoods, too :(

**Comment by jonfroehlich** [comment]
Maybe something to consider for the future? Or would that be hard to implement?

**Comment by jonfroehlich** [comment]
Just happened again. Again, I clicked the 'Feedback' button so that perhaps you could track it?

![image](https://user-images.githubusercontent.com/1621749/54396586-c496a800-4670-11e9-8530-34edc8c47238.png)

**Comment by misaugstad** [comment]
Yeah it would be best to investigate whether these are our fault, or the fault of OSM (it looks like it is our fault).

You remember how we used to have streets that spanned multiple neighborhoods? We don't have that in Newberg, we are just cutting off street segments at the border of neighborhoods. And this could be contributing to the number of tiny segments. We should consider letting streets span multiple neighborhoods again in the future?

**Comment by misaugstad** [comment]
This happened to me once yesterday as well.

**Comment by misaugstad** [comment]
There are other tickets for small street segments. And we have redone how we route users, so anything else that may have been causing this is no longer relevant.
